app/services/optout/OptOutService.scala

```diff
@@ -134,25 +109,51 @@ class OptOutService @Inject()(itsaStatusUpdateConnector: ITSAStatusUpdateConnect
     }
   }
 
-  private def makeOptOutUpdateRequestForOneYear(optOutProposition: OptOutProposition)(implicit
-                                                                                      user: MtdItUser[_],
-                                                                                      shc: HeaderCarrier,
-                                                                                      ec: ExecutionContext): Future[OptOutUpdateResponse] = {
-    val intent = optOutProposition.availableOptOutYears.head
-    makeOptOutUpdateRequest(optOutProposition, intent)
-  }
 
   def makeOptOutUpdateRequest(optOutProposition: OptOutProposition, intent: OptOutTaxYear)(implicit user: MtdItUser[_],
-                                                                                           shc: HeaderCarrier, ec: ExecutionContext): Future[OptOutUpdateResponse] = {
+                                                                                           hc: HeaderCarrier, ec: ExecutionContext): Future[OptOutUpdateResponse] = {
+
 
     val optOutYearsToUpdate = optOutProposition.optOutYearsToUpdate(intent)
 
     val responses: Seq[Future[OptOutUpdateResponse]] = optOutYearsToUpdate.map(optOutYear =>
       itsaStatusUpdateConnector.requestOptOutForTaxYear(optOutYear.taxYear, user.nino, optOutUpdateReason))
     combineByReturningAnyFailureFirstOrAnySuccess(responses)
   }
+
+
+  def nextUpdatesPageOneYearOptOutViewModel()(implicit user: MtdItUser[_], hc: HeaderCarrier, ec: ExecutionContext): Future[Option[OptOutOneYearViewModel]] = {
```

Comment:
this method should be renamed to nextUpdatesPageOptOutViewModel, as it's not specific to one-year opt-out

Reply:
done